packages/cli/src/commands/init.ts

```diff
@@ -51,7 +59,25 @@ export default class Init extends Command {
       })
     }
 
-    fs.cpSync(templateDirectory, fullDirectory, { recursive: true })
+    if (fs.existsSync(fullDirectory) && fs.readdirSync(fullDirectory).length === 0) {
+      fs.rmSync(fullDirectory, { recursive: true })
+    }
+
+    if (!fs.existsSync(fullDirectory)) {
+      fs.mkdirSync(fullDirectory, { recursive: true })
+    }
+
+    const clone = spawnSync('git', ['clone', 'https://github.com/mimic-protocol/init-template.git', fullDirectory], {
+      stdio: 'inherit',
+    })
```

Comment:
We can use something like this to avoid having the error of git not installed

https://www.npmjs.com/package/simple-git